index.js

```diff
@@ -2,6 +2,10 @@
 
 var toString = Object.prototype.toString
 
+if (typeof Buffer === 'undefined') {
+  global.Buffer = require('buffer').Buffer
```

Comment:
Hmm, I'm not really sure that I'm comfortable with patching global variables as a side effect of importing this module.

This module is really meant for environments that already supports `Buffer`, if your environment doesn't, isn't it better to add this line to your project?

Happy to hear more about the problem though!

Reply:
You're right, I patched the environment myself. Thanks for your quick response.